styles.css

```diff
@@ -499,3 +479,529 @@ a:hover {
+  }
+
+  /* last page */
+
+  .about-myself {
+    background-color: #fff;
+    display: flex;
+    border-top-right-radius: 100px;
+    padding-left: 9vw;
+    padding-right: 9vw;
+    padding-top: 142px;
+  }
+
+  .myslef {
+    display: flex;
+    flex-direction: column;
+    max-width: 447px;
+    gap: 12px;
+    flex: 1;
+  }
+
+  .about {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    margin-bottom: initial;
+  }
+
+  .about-myself .text {
+    margin-top: initial;
+    margin-bottom: initial;
+  }
+
+  .about-myself .connect {
+    margin-top: initial;
+    padding-bottom: 12px;
+    padding-top: 8px;
+  }
+
+  .about-myself .social-media {
+    margin-top: initial;
+    margin-bottom: initial;
+  }
+
+  .about-myself .see-project {
+    margin-bottom: initial;
+    margin-top: 22px;
+  }
+
+  .allow-down {
+    width: 36px;
+    height: 18px;
+    padding-right: 10px;
+  }
+
+  .languages-list {
+    margin: initial;
+    margin-right: 36px;
+    flex: 1;
+  }
+
+  .languages-list ul {
+    display: flex;
+    flex-direction: column;
+    width: auto;
+    margin-bottom: 251.85px;
+  }
+
+  .lang-title {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+    display: flex;
+    list-style-type: none;
+    width: auto;
+    padding: 9px;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .land-span {
+    display: flex;
+    gap: 12px;
+  }
+
+  .languages-list .lang {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 8.4vw;
+    background: #f7f7f9;
+    border-radius: 8px;
+    padding: 9px;
+    text-align: left;
+    gap: 16px;
+  }
+
+  .languages-list .lang img {
+    max-width: 48px;
+  }
+
+  .contact-me p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .languages-list .lang p {
+    margin: initial;
+    text-align: left;
+    padding-left: initial;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #253858;
+    padding-bottom: 20px;
+  }
+
+  .frameworks {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+    display: flex;
+    list-style-type: none;
+    width: auto;
+    padding: 9px;
+    align-items: center;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+  }
+
+  .langusge {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+  }
+
+  .sign {
+    padding-right: 20px;
+  }
+
+  /* The contact form  */
+
+  .contact-me {
+    background-color: #6070ff;
+    display: flex;
+    flex-direction: column;
+    border-top-left-radius: 100px;
+    margin-top: -114px;
+    background-image: url("images/contact-form-background-shapes.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 89.6vw 102.86vh;
+    text-align: center;
+  }
```

Comment:
*[OPTIONAL]*

- Hey, please, let's improve our design on the contact section by using the correct font family and reducing the width to match the design as much as we can. See the images below:


*Figma design*
![Screenshot from 2023-02-24 17-18-50](https://user-images.githubusercontent.com/59397159/221231750-c4180c12-8407-4782-9fd3-e81c9a9bee85.png)

*Your design*
![Screenshot from 2023-02-24 17-23-48](https://user-images.githubusercontent.com/59397159/221232128-39f37271-15f7-4b10-86e0-b3ffce188d96.png)

Reply:
Thanks, I am going to fix it right now